config/config-example.js

```diff
@@ -117,6 +117,25 @@ Config.whitelist = [
 	'4cdn\\.org'
 ];
 
+Config.routes = {
+	root: 'pokemonshowdown.com',
+	client: 'play.pokemonshowdown.com',
+	dex: 'dex.pokemonshowdown.com',
+	replays: 'replay.pokemonshowdown.com',
+	usercp: 'pokemonshowdown.com/users'
+};
```

Comment:
I wonder if there's a way to only specify this in one place instead of two.

Maybe have the real config be JSON, and then PHP and the client can both generate config files from that?

Reply:
Sure, that's a good idea.